### PR TITLE
Add email login to demo app

### PR DESCRIPTION
**Pull request by mokagio**
What is says in the title...

This is just an expedient to verify we have workable hooks into the framework for further work, namely adding an SDK-free version of Google SignIn.

To test, launch the app and try to login, you should get a confirmation alert.

<table>
<tr>
	<td>Screen Shot 2022-11-21 at 2 03 00 pm
</td>
	<td>Screen Shot 2022-11-21 at 2 03 03 pm

</td>
	<td>Screen Shot 2022-11-21 at 2 03 23 pm
</td>
	<td>etc...</td>
</table>

---

- [x] I have considered if this change warrants release notes and have added them to the appropriate section in the `CHANGELOG.md` if necessary.

### CODE REVIEW

File: Demo/AuthenticatorDemo.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -215,14 +273,30 @@
...
+				3F27A562291E43F60000996B /* CellConfiguration.swift in Sources */,
+				3F27A556291AE5F00000996B /* Secrets.swift in Sources */,
+				3F27A55C291BC68A0000996B /* UIViewController+Presentation.swift in Sources */,
+				3F27A558291B67C20000996B /* PrintFunctionName.swift in Sources */,
+				3F27A560291E421C0000996B /* ViewController+WordPressAuthenticator.swift in Sources */,
 				3FB5B6F32912420E00F40408 /* ViewController.swift in Sources */,
 				3FB5B6EF2912420E00F40408 /* AppDelegate.swift in Sources */,
 				3FB5B6F12912420E00F40408 /* SceneDelegate.swift in Sources */,
+				3F27A55A291BC6300000996B /* UIAlertController+Utils.swift in Sources */,
+				3F27A55E291E41CC0000996B /* ViewController+WordPressAuthenticationDelegate.swift in Sources */,
```
Comment (mokagio) [inline Demo/AuthenticatorDemo.xcodeproj/project.pbxproj]: I wasn't expecting these to be here...

File: Demo/AuthenticatorDemo/ViewController+WordPressAuthenticationDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+    func shouldPresentSignupEpilogue() -> Bool {
+        true
+    }
+
+    func sync(credentials: AuthenticatorCredentials, onCompletion: @escaping () -> Void) {
+        dismiss(animated: true) { [weak self] in
+            self?.presentAlert(
+                title: "Authentication Successful",
+                message: "Next step will be syncing credentials",
```
Comment (mokagio) [inline Demo/AuthenticatorDemo/ViewController+WordPressAuthenticationDelegate.swift:73]: This is the content of the alert you'll see upon successful email login.

File: Demo/AuthenticatorDemo/Secrets.example.swift [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+@objc
```
Comment (crazytonyli) [inline Demo/AuthenticatorDemo/Secrets.example.swift]: I didn't check, but seeing there are only a few Objective-C files in this repo makes me wondering if this `@objc` is needed.

Comment (mokagio) [reply Demo/AuthenticatorDemo/Secrets.example.swift]: Yes, good point.

I copy pasted this from WordPress iOS, but there's no to keep it verbatim.

Comment (mokagio) [reply Demo/AuthenticatorDemo/Secrets.example.swift]: Addressed in [wordpress-mobile/WordPressAuthenticator-iOS@`487132f` (#706)](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/706/commits/487132f9a642e07e9c6c36337133f19c50108d4b)

**Review by crazytonyli — APPROVED** [review]

**Pull request merged**